galaxy/Dockerfile

```diff
@@ -117,6 +117,7 @@ RUN ansible-playbook /ansible/postgresql_provision.yml && \
     --extra-vars proftpd_use_sftp=True \
     --extra-vars nginx_upload_store_path=/export/nginx_upload_store \
     --extra-vars supervisor_postgres_config_path=$PG_CONF_DIR_DEFAULT/postgresql.conf \
+    --extra-vars supervisor_postgres_autostart=false \
```

Comment:
No, because it should anyway do it as first step:
```
function start_supervisor {
    supervisord -c /etc/supervisor/supervisord.conf
    sleep 5
    {% if supervisor_manage_postgres|bool and not supervisor_postgres_autostart|bool %}
    if [[ $NONUSE != *"postgres"* ]]
    then
        echo "Starting postgres"
        supervisorctl start postgres
    fi
    {% endif %}
```
Instead of starting it during supervisor startup it will start afterwards.